src/main/java/com/xero/api/client/AppStoreApi.java

```diff
@@ -170,6 +170,7 @@ public HttpResponse getSubscriptionForHttpResponse(String accessToken, UUID subs
     }
     HttpHeaders headers = new HttpHeaders();
     headers.setAccept("application/json");
+    headers.setContentType("");
```

Comment:
don't set to blank instead this statement shouldn't be generated

Reply:
removed the empty assignment in all the places